apps/ledger-live-desktop/src/renderer/modals/Send/fields/AmountField.tsx

```diff
@@ -112,11 +111,10 @@ const AmountField = ({
       <RequestAmount
         disabled={!!useAllAmount || walletConnectProxy}
         account={account}
-        validTransactionError={amountError || messageGas}
+        validTransactionError={amountError || messageGas || messageDust}
         validTransactionWarning={amountWarning}
         onChange={onChange}
         value={walletConnectProxy ? transaction.amount : amount}
-        showCountervalue={false}
```

Comment:
Why do we remove this property ?

Reply:
The props is not defined in the children component, did not throw an error before because it was not a tsx file

Reply:
Looks like it's not actually a valid prop of `RequestAmount`